java/src/jmri/implementation/AbstractSignalMast.java

```diff
@@ -123,6 +123,23 @@ public void setHeld(boolean newHeld) {
         }
     }
 
+    public boolean isAtStop() {
+        if  (speed.equals("0")) return true;  // should this also include DANGER?
```

Comment:
I'm merging this PR now. But it might be a good idea to ask about this on the JMRI developers list to get it right.